### PR TITLE
Edited installing swift index 

### PR DESCRIPTION
**Pull request by RobinBateman808**
### Motivation:

Help developers to get started with documentation.

### Modifications:

Added user-friendly language and simple steps for clarity.

### Result:

Provide new users and developers with an instructional vs functional approach.

### CODE REVIEW

File: install/index.md [hunk 1]
```diff
@@ -3,9 +3,14 @@ layout: page
...
 ---
 
-Follow the instructions below to install the latest version of Swift on a [supported platform](/platform-support).
+Whether you are a beginner or an experienced programmer, learning and using Swift can open up opportunities for you to build innovative applications for Apple devices and applications beyond Apple's ecosystem. Not only can you use Swift on macOS, but using Swift on Linux and Windows makes Swift a valuable language for developers looking to work across different platforms and environments. 
 
-You can also [download nightly snapshots and older releases](/download).
+Follow these general steps to start using Swift:
+
+1. Choose your development platform below and follow the instructions to install the latest version of Swift. See [supported platform](/platform-support) for the level of help for each platform.
+2. After the installation is complete, you can verify the installation by running the swift —version command.
```
Comment (0xTim) [inline install/index.md:11]:
```suggestion
2. After the installation is complete, you can verify the installation by running `swift —version`.
```

File: install/index.md [hunk 2]
```diff
@@ -3,9 +3,14 @@ layout: page
 title: Install Swift
 ---
 
-Follow the instructions below to install the latest version of Swift on a [supported platform](/platform-support).
+Whether you are a beginner or an experienced programmer, learning and using Swift can open up opportunities for you to build innovative applications for Apple devices and applications beyond Apple's ecosystem. Not only can you use Swift on macOS, but using Swift on Linux and Windows makes Swift a valuable language for developers looking to work across different platforms and environments. 
 
-You can also [download nightly snapshots and older releases](/download).
+Follow these general steps to start using Swift:
+
+1. Choose your development platform below and follow the instructions to install the latest version of Swift. See [supported platform](/platform-support) for the level of help for each platform.
```
Comment (0xTim) [inline install/index.md:10]:
```suggestion
1. Choose your development platform below and follow the instructions to install the latest version of Swift. See the [supported platforms page](/platform-support) for the platforms that Swift supports.
```

**Review by 0xTim — COMMENTED** [review]
Couple of suggestions

File: install/index.md (hunk 1 above)
Comment (RobinBateman808) [reply install/index.md:11]: good catch!

File: install/index.md (hunk 2 above)
Comment (RobinBateman808) [reply install/index.md:10]: This is a little redundant. Maybe say something as simple as, "See the [supported platforms page](/platform-support) for more information." wdyt?

**Comment by RobinBateman808** [comment]
Hi @shahmishal - please let me know if you have any changes. If it lgty, feel free to merge. Thanks!

**Comment by 0xTim** [comment]
@swift-ci please test

**Comment by RobinBateman808** [comment]
@0xTim, @shahmishal - thank you!

**Review by shahmishal — APPROVED** [review]

**Pull request merged**